src/panel/common/commonActions.js

```diff
@@ -54,10 +76,27 @@ function isInvalidTags(tagArr) {
 	return false;
```

Comment:
For trimming, the tags should be trimmed by `tagStringToArray` already I think
I didn't get what you mean by placing `indicateInvalidTitleOrTags()` & `indicateInvalidId()` in `addNewList` 😅